README.md

```diff
@@ -47,29 +47,24 @@ metadata:
 
 This solution requires Flux CLI locally and Flux Controller on your Kubernetes cluster. Flux requires access to a source repository via api and access to the kubernetes cluster you want to use for testing. Please follow the below steps for installing these pre-requisites.
 
-If you do not alreday have access to a running kubernetes cluster you can consider a local easy to deploy setup such as [k3s](https://k3s.io/) or you may choose a hosted service such as [AWS EKS](https://aws.amazon.com/eks/)
+If you do not already have access to a running kubernetes cluster you can consider setting up a local [k3s](https://k3s.io/) cluster or you may choose a hosted service such as [AWS EKS](https://aws.amazon.com/eks/).
```

Comment:
Let's add https://anywhere.eks.amazonaws.com/docs/getting-started/local-environment/ to the list for local dev. I think we should start with it and call out k3s briefly if at all.